r/rsparkling/R/h2o_context.R

```diff
@@ -17,11 +17,10 @@ h2o_context.spark_connection <- function(x, strict_version_check = TRUE) {
 }
 
 #' @export
-h2o_context.spark_jobj <- function(x) {
-  h2o_context(spark_connection(x))
+h2o_context.spark_jobj <- function(x, strict_version_check = TRUE) {
+  h2o_context.spark_connection(spark_connection(x), strict_version_check=strict_version_check)
```

Comment:
because we already convert `x` with `spark_connection()` we can explicitly direct to `h2o_context.spark_connection` method, this avoid extra dispatching via `h2o_context`